packages/gatsby-source-filesystem/src/error-utils.js

```diff
@@ -0,0 +1,26 @@
+export const CODES = {
+  Generic: `10000`,
+  MissingResource: `10001`,
+}
+
+export const pluginPrefix = `gatsby-source-filesystem`
+
+export function prefixId(id) {
+  return `${pluginPrefix}_${id}`
+}
+
+// TODO: Refactor to use contextual data instead of only context.sourceMessage
+// once reporter.setErrorMap is guaranteed to be available
+export const ERROR_MAP = {
+  [CODES.Generic]: {
+    text: context => `FileSystem: ${context.sourceMessage}`,
+    level: `ERROR`,
+    type: `PLUGIN`,
+  },
+  [CODES.MissingResource]: {
+    text: context => `FileSystem: ${context.sourceMessage}`,
```

Comment:
```suggestion
    text: context => context.sourceMessage,
    level: `ERROR`,
    type: `PLUGIN`,
  },
  [CODES.MissingResource]: {
    text: context => context.sourceMessage,
```

We won't need the message prefix as this was only added in `gatsby-plugin-page-creator` as it's an internal plugin used by default and it makes it easier to see where it's coming from.

This plugin is something you explicitly add and the ID will also have the prefix, so we can do it like this 👍